capstone-frontend/src/components/LeftSideBar.jsx

```diff
@@ -1,43 +1,92 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import '../App.css';
+import { Accordion, Card } from 'react-bootstrap'
+import "../styles/LeftSideBar.css"
+// import '../App.css';
```

Comment:
Do we even need ```App.css``` here in this file? If not I think we can just delete it. I don't think we use it anymore.

Reply:
We don't need it here, but in other components yes.